web/cypress/integration/service/edit-service-with-upstream.spec.js

```diff
@@ -25,8 +25,7 @@ context('Edit Service with Upstream', () => {
   });
 
   it('should create a test upstream', function () {
-    cy.visit('/');
-    cy.contains('Upstream').click();
+    cy.visit('/upstream/list');
```

Comment:
Is it necessary to add `cy.get(this.domSelector.empty).should('be.visible');` to make sure the upstream list rendered well just like https://github.com/apache/apisix-dashboard/pull/1645/files#diff-e5538d6ea581a815051ee7ba013803f5b5c795e4979f908e3aa8c51844521a36R34 ?

Reply:
Agree with you